packages/gamut/src/ToolTip/index.tsx

```diff
@@ -1,15 +1,162 @@
-import cx from 'classnames';
+import { fontSmoothing, pxRem } from '@codecademy/gamut-styles';
+import styled from '@emotion/styled';
 import React, { ReactNode } from 'react';
 
 import { VisualTheme } from '../theming/VisualTheme';
-import styles from './styles.module.scss';
 
-export enum ToolTipPosition {
-  BottomLeft = 'bottom-left',
-  BottomRight = 'bottom-right',
-  TopLeft = 'top-left',
-  TopRight = 'top-right',
-}
+export type ToolTipPosition =
+  | 'bottom-left'
+  | 'bottom-right'
+  | 'top-left'
+  | 'top-right';
+
+const arrowWidth = `1rem`;
+const arrowHeight = `0.5rem`;
+const arrowPaddingHorizontal = `1.5rem`;
+const arrowPaddingVertical = `0.25rem`;
+
+const containerOffsetHorizontal = `calc(100% - (${arrowWidth} / 2) + ${arrowPaddingHorizontal})`;
+const containerOffsetVertical = `calc(100% + ${arrowHeight} + ${arrowPaddingVertical})`;
+
+const arrowOffsetHorizontal = `calc(${arrowPaddingHorizontal} - (${arrowWidth} / 2))`;
+const arrowOffsetVertical = `calc(-${arrowHeight} / 2)`;
+
+const slightShadow = `rgba(0, 0, 0, 0.15)`;
+
+const TooltipWrapper = styled.div`
+  position: relative;
+  display: inline-flex;
+`;
+
+const TargetContainer = styled.div`
+  cursor: pointer;
+  border: 0;
+  background: none;
+  padding: 0;
+`;
+
+type ToolTipContainerProps = {
+  position: ToolTipPosition;
+  variant: VisualTheme;
+};
+
+const ToolTipContainer = styled.div<ToolTipContainerProps>`
+  ${fontSmoothing()}
+
+  display: flex;
+  opacity: 0;
+  transition: opacity 0ms;
+  transition-delay: 750ms;
+  position: absolute;
+  width: 16rem;
+  visibility: hidden;
+
+  ${({ theme, variant }) =>
+    variant === 'dark' && `background: ${theme.colors.black};`}
+
+  // Both before and after psuedo-elements are used because ::after's background should go over the container's
+  // and ::before's box-shadow should be behind the container itself
+  &::after,
+  &::before {
+    content: '';
+    display: block;
+    height: ${arrowHeight};
+    position: absolute;
+    transform: rotate(45deg);
+    width: ${arrowHeight};
+  }
+
+  &::after {
+    background: ${({ theme, variant }) =>
+      variant === 'dark' ? theme.colors.black : theme.colors.white};
```

Comment:
Since we should be covered by https://github.com/Codecademy/client-modules/pull/1357/files#diff-e8dca7b1934565fd888dec9443a2f38e5c53edc1b2c6487dc13e157a0b4456d1R54 :)

```suggestion
    background:  inherit;
```